docs/src/main/sphinx/admin/properties-writer-scaling.md

```diff
@@ -41,21 +41,22 @@ session property.
 - **Type:** {ref}`prop-type-integer`
 - **Default value:** The number of physical CPUs of the node with a maximum of 32
 
-Maximum number of concurrent writers per task upto which the task can be scaled when
-`task.scale-writers.enabled` is set. Increasing this value may improve the
-performance of writes when the query is bottlenecked on writing. Setting this too high
-may cause the cluster to become overloaded due to excessive resource utilization.
+Maximum number of concurrent writers per task up to which the task can be scaled
+when `task.scale-writers.enabled` is set. Increasing this value may improve the
+performance of writes when the query is bottlenecked on writing. Setting this
+too high may cause the cluster to become overloaded due to excessive resource
+utilization.
 
 ## `writer-min-size`
 
 - **Type:** {ref}`prop-type-data-size`
 - **Default value:** `32MB`
+- **Session property:** `writer_min_size`
 
 The minimum amount of data that must be written by a writer task before
 another writer is eligible to be added. Each writer task may have multiple
 writers, controlled by `task.writer-count`, thus this value is effectively
-divided by the number of writers per task. This can be specified on a
-per-query basis using the `writer_min_size` session property.
+divided by the number of writers per task.
 
 :::{warning}
 This property is deprecated now. Please use `writer-scaling-min-data-processed`.
```

Comment:
@raunaqmorarka .. should we remove docs for this property?

Reply:
yes, shouldn't be documented anymore, we can just mention it's replaced by writer-scaling-min-data-processed

Reply:
Done